src/build/dotnet/steps/nuget-package.steps.yaml

```diff
@@ -13,25 +13,61 @@ steps:
       skip: ${{ parameters.skipDependencyChecks }}
       continueIfVulnerabilities: true
 
-  - template: /src/build/dotnet/tasks/netcore/version.yaml
-    parameters:
-      releaseBranchName: ${{parameters.releaseBranchName}}
-      packageSource: ${{parameters.packageSource}}
+  - ${{ if eq(parameters.applyVersioning, true) }}:
+  - task: UpdateDotNetVersions@12
```

Comment:
This task needs indenting

Reply:
Please can you check other indentation issues, particularly under these `if` expressions

Reply:
yeah I tried using a yaml validator which told me these were the correct indents, im assuming the extension is just wrong in this case so ive now amended them all to what I think they should be, if they are still wrong please let me know